examples/tensorflow/text-classification/run_text_classification.py

```diff
@@ -398,10 +358,12 @@ def main():
             sentence1_key, sentence2_key = non_label_column_names[:2]
         else:
             sentence1_key, sentence2_key = non_label_column_names[0], None
-
     # Padding strategy
     if data_args.pad_to_max_length:
         padding = "max_length"
+    # elif isinstance(training_args.strategy, tf.distribute.TPUStrategy):
+    #    logger.warning("Forcing --pad_to_max_length because we're running on TPU!")
+    #     padding = "max_length"
```

Comment:
Why commented out?

Reply:
This code is a leftover from when we padded the entire dataset into one dense Tensor. This is the least memory-efficient option of all, and we don't need to do it even on TPU. Instead, we can just ensure that we pad batches to the same size each time, which is now handled in the Dataset logic. I've removed it!